client/src/lib/components/Tag/Tag.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+  import { createEventDispatcher } from "svelte";
+
+  const dispatch = createEventDispatcher();
```

Comment:
(Nit) Suggestion qu'on type le dispatcher ici également

```suggestion
  const dispatch = createEventDispatcher<{ click: Tag }>();
```